src/main/java/seedu/address/logic/commands/UpdateCommand.java

```diff
@@ -132,10 +132,10 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         MeetingTime updatedMeetingTime = editPersonDescriptor.getMeetingTime().orElse(personToEdit.getMeetingTime());
         Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
         FilePath updatedFilePath = personToEdit.getFilePath();
-        Remark updatedRemark = personToEdit.getRemark(); // edit command does not allow editing remarks
+        Description updatedDescription = personToEdit.getDescription(); // edit command does not allow editing remarks
```

Comment:
I think we should enhance UpdateCommand to allow updating of a client's description.
Can implement as a separate PR/Issue

Reply:
Current implementation does not allow updating of descriptions

Reply:
right now editting description has its own command